### PR TITLE
Add API

### PR DESCRIPTION
**Pull request by bohaz**
> ## In this pull request

- [ ] Make use of the [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) as your backend.
- [ ] Use [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) to make your HTTP requests to the backend
- [ ] Fetch your `books` from the Bookstore API after the component has rendered
- [ ] Refactor your add book and remove book features to persist your changes in the server
- [ ] Your application should behave the same way after the refactoring

### Happy review!!

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
+import api from '../../api';
 
-const initialState = {
-  books: [],
-};
+export const addBookAsync = createAsyncThunk(
+  'books/addBookAsync',
+  async ({ APP_ID, book }) => {
+    try {
+      const response = await api.post(`/apps/${APP_ID}/books`, book);
```
Comment (brytebee) [inline src/redux/books/booksSlice.js:9]: Adding a book should push a book to the API and removing a book should remove the book from the API.

Kindly verify that the request correctly posts a book to the endpoint. On page load, fetch the books and render them to the page.

The response should be set to the store so you can access it from the BookList component as you've defined with `setBook` `useState` function.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @bohaz 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/booksSlice.js (hunk 1 above)
Comment (bohaz) [reply src/redux/books/booksSlice.js:9]: @brytebee Hi, that's what my app does. In my dev tool, a new book is created when I click on add and is removed when I click on remove button. 

Captura de pantalla (10)

Captura de pantalla (11)

Captura de pantalla (12)

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @bohaz , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  App works as required :+1: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**